### PR TITLE
Section: Continuous collaborative manuscript drafting

### PR DESCRIPTION
**Pull request by dhimmel**
@agitter and @cgreene sorry for getting this in last minute.

I used [semantic linefeeds](http://rhodesmill.org/brandon/2012/one-sentence-per-line/) (one sentence per line), since I think they're better than word wrap. I'm sure codeclimate will have a fit.

### CODE REVIEW

**Comment by agitter** [comment]
This will have merge conflicts with #433.  Should we merge #433 to master and then update this as needed?

**Comment by dhimmel** [comment]
> This will have merge conflicts with #433. Should we merge #433 to master and then update this as needed?

As the pull request author, it's my responsibility to resolve conflicts and keep the PR mergeable based on whatever the current master is. So proceed and I'll deal with any conflicts (:

**Comment by dhimmel** [comment]
I hadn't actually seen #433. Looking at the [changes](https://github.com/greenelab/deep-review/pull/433/files), I'd merge it #433 first so I can make any changes here in the context of #433.

**Comment by agitter** [comment]
> I hadn't actually seen #433. Looking at the changes, I'd merge it #433 first so I can make any changes here in the context of #433.

Agreed.  I will merge #433 because @cgreene approved it previously and I have finished collecting acknowledgements.  I'll open another PR later to add a funding statement.

**Comment by agitter** [comment]
@dhimmel Merged #433.  Let me know when you've resolved the conflicts and I'll review this.

**Comment by dhimmel** [comment]
> Let me know when you've resolved the conflicts and I'll review this.

@agitter ready

File: sections/07_conclusions.md [hunk 1]
```diff
@@ -91,22 +91,41 @@ very large collections of input data into interpretable models that spur
...
-did participate, are acknowledged at the end of the manuscript.
-
-`TODO: update after finalizing discussion in #369`
+We created an open repository on the GitHub version control platform ([`greenelab/deep-review`](https://github.com/greenelab/deep-review)) [@url:https://github.com/greenelab/deep-review].
+Here, we engaged with numerous authors from papers within and outside of the area.
+The manuscript was drafted via GitHub commits by 25 individuals who met the ICJME standards of authorship.
+These were individuals who contributed to the review of the literature;
+drafted the manuscript or provided substantial critical revisions;
+approved the final manuscript draft; and agreed to be accountable in all aspects of the work.
+Individuals who did not contribute in all of these ways, but who did participate, are acknowledged at the end of the manuscript.
```
Comment (agitter) [inline sections/07_conclusions.md]: This was leftover from my pull request, but does "are acknowledged below." sound better?  The acknowledgements immediately follow.

File: sections/07_conclusions.md [hunk 2]
```diff
@@ -91,22 +91,41 @@ very large collections of input data into interpretable models that spur
...
-numerous authors from papers within and outside of the area. The manuscript was
-drafted via GitHub commits by 25 individuals who met the ICJME standards
-of authorship. These were individuals who contributed to the review of the
-literature; drafted the manuscript or provided substantial critical revisions;
-approved the final manuscript draft; and agreed to be accountable in all aspects
-of the work. Individuals who did not contribute in all of these ways, but who
-did participate, are acknowledged at the end of the manuscript.
-
-`TODO: update after finalizing discussion in #369`
+We created an open repository on the GitHub version control platform ([`greenelab/deep-review`](https://github.com/greenelab/deep-review)) [@url:https://github.com/greenelab/deep-review].
```
Comment (agitter) [inline sections/07_conclusions.md]: I'm not sure how `greenelab/deep-review` will be rendered by the journal.  For the preprint and initial review we can do what we want I suppose.

File: sections/07_conclusions.md [hunk 3]
```diff
@@ -91,22 +91,41 @@ very large collections of input data into interpretable models that spur
...
+Hence, diverse expertise was required to provide a forward-looking perspective.
+Accordingly, we collaboratively wrote this review in the open, enabling anyone with expertise to contribute.
+We wrote the manuscript in markdown and tracked changes using git.
+Contributions were handled through GitHub, with individuals submitting "pull requests" to suggest additions to the manuscript.
+
+To facilitate citation, we [defined](https://github.com/greenelab/deep-review/blob/master/CONTRIBUTING.md#markdown) a markdown citation syntax.
+We supported citations to the following identifier types (in order of preference): DOIs, PubMed IDs, arXiv IDs, and URLs.
+References were automatically generated from citation metadata by querying APIs to generate [Citation Style Language](http://citationstyles.org/) (CSL) JSON items for each reference.
+[Pandoc](http://pandoc.org/) and [pandoc-citeproc](https://github.com/jgm/pandoc-citeproc) converted the markdown to HTML and PDF, while rendering the formatted citations and references.
+In total, referenced works consisted of 274 DOIs, 4 PubMed records, 106 arXiv preprints, and 42 webpages.
```
Comment (agitter) [inline sections/07_conclusions.md]: Is it worth noting that these webpages include manuscripts and webpages?

File: sections/07_conclusions.md [hunk 4]
```diff
@@ -91,22 +91,41 @@ very large collections of input data into interpretable models that spur
...
 
+### Continuous collaborative manuscript drafting
+
+We recognized that deep learning in precision medicine is a rapidly developing area.
+Hence, diverse expertise was required to provide a forward-looking perspective.
+Accordingly, we collaboratively wrote this review in the open, enabling anyone with expertise to contribute.
+We wrote the manuscript in markdown and tracked changes using git.
+Contributions were handled through GitHub, with individuals submitting "pull requests" to suggest additions to the manuscript.
+
+To facilitate citation, we [defined](https://github.com/greenelab/deep-review/blob/master/CONTRIBUTING.md#markdown) a markdown citation syntax.
```
Comment (agitter) [inline sections/07_conclusions.md]: Including hyperlinks in this section is a stylistic departure from the rest of the manuscript.  It stands out to me.  Converting to URL citations is less functional, but consistent with that we've done elsewhere.  What do you think?

File: sections/07_conclusions.md [hunk 5]
```diff
@@ -91,22 +91,41 @@ very large collections of input data into interpretable models that spur
...
-control system [@url:https://github.com/greenelab/deep-review] and engaged with
-numerous authors from papers within and outside of the area. The manuscript was
-drafted via GitHub commits by 25 individuals who met the ICJME standards
-of authorship. These were individuals who contributed to the review of the
-literature; drafted the manuscript or provided substantial critical revisions;
-approved the final manuscript draft; and agreed to be accountable in all aspects
-of the work. Individuals who did not contribute in all of these ways, but who
-did participate, are acknowledged at the end of the manuscript.
-
-`TODO: update after finalizing discussion in #369`
```
Comment (agitter) [inline sections/07_conclusions.md]: Please restore this `TODO`.  Once we establish the final author ordering, which will likely include some random element, I would like to describe that process here.

File: sections/07_conclusions.md [hunk 6]
```diff
@@ -91,22 +91,41 @@ very large collections of input data into interpretable models that spur
 scientists to ask questions that they didn't know how to ask, it will be clear
 that deep learning has transformed biology and medicine.
 
+### Continuous collaborative manuscript drafting
```
Comment (agitter) [inline sections/07_conclusions.md]: The transition from the conclusions to this section feels abrupt.  We close with our big punchline at the end of the conclusion and then move to this methods-like section.

Would moving it below the author contributions or acknowledgements help signal that we've moved to a new section of the manuscript?  I'm not sure of any better places to put it.

**Comment by dhimmel** [comment]
@agitter thanks for the review. Will address the issues tomorrow.

File: sections/07_conclusions.md (hunk 1 above)
Comment (agapow) [inline sections/07_conclusions.md]: "are acknowledged below." does sound better

File: sections/07_conclusions.md (hunk 4 above)
Comment (dhimmel) [inline sections/07_conclusions.md]: I use URLs knowing that they will likely be removed by the journal. However, there is still value added in the interim. I will make sure some of these URLs shouldn't also be references. So the hyperlink policy is that they the text is sufficient without them. Do you think that's a good solution?

File: sections/07_conclusions.md (hunk 2 above)
Comment (dhimmel) [inline sections/07_conclusions.md]: preformatted text should be within the standard typesetting toolkit. Worst case journal removes hyperlink and code-formatting, but then we still have the citation.

File: sections/07_conclusions.md (hunk 6 above)
Comment (agitter) [inline sections/07_conclusions.md]: I like that idea.  Should we keep `Author contributions` and `Acknowledgements` in `07_conclusions.md` or move those to a new `09_contributions`?  I think having Methods follow Conclusions directly works well if it is prefaced by a `## Methods` subheading.

File: sections/07_conclusions.md (hunk 4 above)
Comment (agitter) [inline sections/07_conclusions.md]: Yes

File: sections/07_conclusions.md (hunk 6 above)
Comment (dhimmel) [inline sections/07_conclusions.md]: See what I did in https://github.com/greenelab/deep-review/pull/477/commits/11276830059ba1f740520ac1bfeb7cd54332fb7c. I'm open to other rearrangements like what @agitter suggested if there's a strong preference. Let's deal with that in another PR if desired. Will merge this now.

**Pull request merged**